index.html

```diff
@@ -232,39 +235,488 @@ <h3>Create</h3>
 the public key value and expanding it into DID Document. An example is given
 below that expands the ed25519 <tt>did:key</tt>
-<code>did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH</code> into its
+<code>did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2</code> into its
 associated DID Document:
         </p>
 
         <pre class="example" title="A DID Document derived from a did:key">
           {
             "@context": [
               "https://www.w3.org/ns/did/v1",
-              "https://w3id.org/security/suites/ed25519-2018/v1",
-              "https://w3id.org/security/suites/x25519-2019/v1"
+              "https://w3id.org/security/suites/ed25519-2020/v1",
+              "https://w3id.org/security/suites/x25519-2020/v1"
             ],
             "id": "did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2",
             "verificationMethod": [
               {
                 "id": "did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2",
-                "type": "Ed25519VerificationKey2018",
+                "type": "Ed25519VerificationKey2020",
                 "controller": "did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2",
-                "publicKeyBase58": "5ANg2DZ4jk7VGtHHsv3SGKjvi79WHvfNp9L6woYPqQqe"
+                "publicKeyMultibase": "z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2"
               },
               {
-                "id": "did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6LSni4ALZW58RuBPizpNeTK3HGfZfSuE3zcxeBubnrssYuh",
-                "type": "X25519KeyAgreementKey2019",
+                "id": "did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#zC2szpFhD2yBSJLd3qzwMih4BiWunXSpU5fUE7LDMAB8w",
+                "type": "X25519KeyAgreementKey2020",
                 "controller": "did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2",
-                "publicKeyBase58": "C2szpFhD2yBSJLd3qzwMih4BiWunXSpU5fUE7LDMAB8w"
+                "publicKeyMultibase": "zC2szpFhD2yBSJLd3qzwMih4BiWunXSpU5fUE7LDMAB8w"
               }
             ],
             "assertionMethod": ["did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2"],
             "authentication": ["did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2"],
             "capabilityInvocation": ["did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2"],
             "capabilityDelegation": ["did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2"],
-            "keyAgreement": ["did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#z6LSni4ALZW58RuBPizpNeTK3HGfZfSuE3zcxeBubnrssYuh"]
+            "keyAgreement": ["did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2#zC2szpFhD2yBSJLd3qzwMih4BiWunXSpU5fUE7LDMAB8w"]
```

Comment:
Same issue here -- the hash fragment should start with `z6LS`.

Reply:
Fixed in 51082d03db60678a0c7d84dfc1cbdfb5fb9b83c6.